Library/src/main/java/BookService.java

```diff
@@ -0,0 +1,109 @@
+import java.io.FileWriter;
+import java.io.IOException;
+public class BookService {
+    // to jest model
+    static boolean isWorking = true;
+    public static void doSomethingWithIt(String input) {
+        while (isWorking) {
+            switch (input) {
+                case "1":
+                    input = "0";
+                    showTitles();
+                    break;
+                case "2":
+                    input = "0";
+                    BookController.addBook();
+                    break;
+                case "3":
+                    input = "0";
+                    bookRemoval();
+                    break;
+                case "4":
+                    input = "0";
+                    viewInfo();
+                    break;
+                case "5":
+                    input = "0";
+                    saveInFile();
+                    break;
+                case "6":
+                    input = "0";
+                    isWorking = false;
+                    break;
+                default:
+                    input = "0";
+                    BookView.getMad();
+                    break;
+            }
+        }
+    }
+    public static void showTitles() {
+        for (int i = 0; i<BookRepository.listOfBooks.size(); i++) {
+            BookView.printTitles(i);
+        }
+        BookView.showMenu();
+        BookController.askAndGive();
+    }
+    public static void createNewBook(String title, String author, String genre, String descrip) {
+        BookRepository.listOfBooks.add(new Book(title, author, genre, descrip));
+        BookView.showMenu();

```

Comment:
Trochę mieszasz warstwy. W takich sytuacjach, serwis ma tylko zwracać dane i wykonywać polecenia kontrolerów.

Wyobraź sobie to piekło, gdy będziesz edytował widok, i nagle dostaniesz 150 powiązań z serwisu